kolibri_explore_plugin/assets/src/components/InstallContentModal.vue

```diff
@@ -73,156 +53,111 @@
 
   import client from 'kolibri.client';
   import urls from 'kolibri.urls';
-  import ProgressDownloadIcon from 'vue-material-design-icons/ProgressDownload.vue';
+
+  const UPDATE_DELAY = 1500;
 
   export default {
     name: 'InstallContentModal',
-    components: {
-      ProgressDownloadIcon,
-    },
-    emits: ['hide', 'showModal', 'newContent'],
-    props: {
-      visible: {
-        type: Boolean,
-        default: false,
-      },
-      collection: {
-        type: Object,
-        default: null,
-      },
-      grade: {
-        type: String,
-        default: 'primary',
-      },
-    },
+    components: {},
+    emits: ['completed'],
+    props: {},
     data() {
       return {
-        pollingId: null,
-        jobs: null,
-        channelsDownloaded: 0,
-        downloading: false,
+        status: null,
+        updateIntervalId: null,
       };
     },
     computed: {
-      progress() {
-        const jobs = this.jobs.map(j => {
-          if (j.percentage > 1 || !j.channel_name) {
-            return 0;
+      titleLabel() {
+        if (this.status !== null) {
+          if (this.status.stage === 'IMPORTING_CONTENT') {
+            return 'Downloading…';
```

Comment:
There are a lot of strings here, so it would be nice if they were going through the localization system so we don't need to find them all later :) [`ContentUnavailablePage`](https://github.com/endlessm/kolibri-explore-plugin/blob/master/kolibri_explore_plugin/assets/src/views/ContentUnavailablePage.vue) has a nice straightforward example of all we need there for the moment.

```suggestion
            return this.$tr('titleLabelDownloading');
```

Reply:
Excellent. Yes I'm happy to make this a new policy for upcoming frontend changes. All labels should use the translation system.